packages/app/src/app/overmind/effects/keybindingManager.ts

```diff
@@ -306,21 +32,6 @@ export default {
       binding => binding.bindings && binding.bindings.filter(Boolean).length
     );
   },
-  start() {
-    if (
-      isStarted ||
-      _overmindInstance.state.preferences.settings.experimentVSCode
-    ) {
-      return;
-    }
-
-    window.addEventListener('keydown', onKeyDown);
-    window.addEventListener('keyup', onKeyUp);
-    isStarted = true;
-  },
-  pause() {
-    window.removeEventListener('keydown', onKeyDown);
-    window.removeEventListener('keyup', onKeyUp);
-    isStarted = false;
-  },
+  start() {},
```

Comment:
Since `_overmindInstance.state.preferences.settings.experimentVSCode` is always `true`, we'll always instantly return.
So we don't need the body at all